app/src/main/java/io/zenandroid/onlinego/data/model/ogs/OGSPlayer.kt

```diff
@@ -1,24 +1,28 @@
 package io.zenandroid.onlinego.data.model.ogs
 
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import io.zenandroid.onlinego.data.model.local.Player
 
 /**
  * Created by alex on 04/11/2017.
  */
+@Entity
```

Comment:
Pls don't put OGSPlayer in the database, put Player instead.

Reply:
This actually already was removed from the DB, just needed to drop the Entity annotation